### ISSUE TITLE
please tell me these where not used in training.... some are really insufficient and lead to issues

### ISSUE THREAD

**Issue by tampadesignr** [EliSchwartz/imagenet-sample-images#3]
please tell me these where not used in training.... some are really bad and lead to issues
i am doing research from the perspective of artist building the system... and the labeling is insufficient
do you have anymore data that is the actual used base data images and labels..
i am having the ai reverse engineer its original documents kinda  : )

**Comment by EliSchwartz** [comment]
Unfortunately, those are indeed from the imagenet training set. There have been later efforts to create a cleaner version of the test set, but I don’t think there is a better version for the train set.

**Comment by tampadesignr** [comment]
there is a label for ear
there are two human ears and a thing of corn (ear)
100% sure visual that mucked something ( after using lora's and modified base models )-- visual ques only master artist would see
and using the base --untouched -- see things are are simple fix not needing 50000, images of training running for 17 hours
-- i am gathering as much info to creating a lora training config/ make a sdxl base (built from a visual artist perspective with high level computer science understanding -- creating a communication method closer to what the intention was in the first place
-- finding info is very difficult --- manual from a -z of everything under the hood written so anybody could understand : )  talking about the thing that apparently doesn't exist
-- any help pdf manual etc....
-- doing my homework as much as i can before... creating
thank you
what was the order of information taught, how was it confirmed, what master lever visual artist helped in the process to confirm concepts etc...?
